modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -396,7 +399,7 @@ function editConsentStatusFields($db, $user)
     // Get CandID
     $candIDParam = $_POST['candID'];
     $candID      = (isset($candIDParam) && $candIDParam !== 'null') ?
-        $candIDParam : null;
+        new CandID($candIDParam) : null;
 
     $candidate   = \Candidate::singleton($candID);
```

Comment:
Would you be able to modify this file to return a `400` response when `$_POST['candID']` is not set?

It's not possible to pass `null` into `Candidate::singleton()`. I think it will generate a TypeError in that case.

It may be an edge-case but it would be nice to handle it properly.

Reply:
@johnsaigle Makes sense. Done!